plugins/woocommerce/src/Admin/API/Options.php

```diff
@@ -220,6 +220,7 @@ public static function get_default_option_permissions() {
 			'woocommerce_admin_customize_store_completed',
 			'woocommerce_admin_customize_store_completed_theme_id',
 			'woocommerce_admin_customize_store_survey_completed',
+			'woocommerce_admin_launch_your_store_survey_completed',
```

Comment:
I don't have a strong preference, but I think we should be careful updating Options.php as it is deprecated. 

I ended up creating a new API endpoint for LYS in https://github.com/woocommerce/woocommerce/pull/46149/files#diff-f73e4780f54b17eaf799454a972d6f659f8d624638930cdaed31fbfc9b521621

What do you think we use the new endpoint?

Reply:
I didn't realize this class was deprecated, good callout here. These options are gathered using the Options data store, along with the rest of the Coming soon related options and added here so they are made available. Maybe there is a hook somewhere else to make sure these options are preloaded? It would be a good follow up PR I think.

Reply:
> I don't have a strong preference, but I think we should be careful updating Options.php as it is deprecated.
> I ended up creating a new API endpoint for LYS in https://github.com/woocommerce/woocommerce/pull/46149/files#diff-f73e4780f54b17eaf799454a972d6f659f8d624638930cdaed31fbfc9b521621

> What do you think we use the new endpoint?

@moon0326 Thanks for catching this and for the example! Yes, I think we should have a new API. However, I would like to add it in your class. In the meantime, we can either leave the option update in this class and make a follow-up, or we could also wait for your PR to be merged. Does either sounds ok to you?

>  Maybe there is a hook somewhere else to make sure these options are preloaded? It would be a good follow up PR I think.

@psealock I don't recall if we have a hook to preload options, but I believe we could simply run a `wp.select` query when we load LYS screen to initiate pre-loading of relevant options. What do you think?

Reply:
@psealock @ilyasfoo Yup! A follow-up PR sounds good 👍

Reply:
>I believe we could simply run a wp.select query when we load LYS screen to initiate pre-loading of relevant options. What do you think?

The `use-launch-your-store` component will do that in any case. Now that I think about it, this actually isn't about pre-loading but instead about permissions. If options aren't on this list, an access error occurs. In any case, good to further investigate in a follow up.